src/lex.js

```diff
@@ -146,7 +149,7 @@ module.exports = function(script) {
                 }
             }
 
-            tokens.push(new types.IntValue(digits));
+            tokens.push(new tokenTypes.Literal('integer', digits));
```

Comment:
Well, `+10000000000000000` is currently an invalid literal. Should be detected as early as possible, probably. But it's a minor issue as either way it's detected during compilation.